nixos/modules/rename.nix

```diff
@@ -61,6 +61,8 @@ with lib;
       to handle FIDO security tokens, so this isn't necessary anymore.
     '')
 
+    (mkRemovedOptionModule [ "services" "seeks" ] "The seeks module has been removed")
```

Comment:
this is going to read redundantly to the end user https://github.com/NixOS/nixpkgs/blob/master/lib/modules.nix#L644, this should be a replacement instruction. Which should be that there is none.
(btw, I doubt anyone is using this is any kind of production because how abandoned it is, but for the sake of practice we should still do it)

Reply:
Addressed in 2e0542ebacb2b8cb938d848ab743ca9df702f56e